### ISSUE TITLE
Member profile page. Add Email button

### ISSUE THREAD

**Issue by ianp5a** [GetTogetherComm/GetTogether#228]
Requirement:  On the users Profile page there should be a Contact/Email possibility like the one in the Team Members List. Where there is an email (Envelope) button next to each name in the list.

Reason: This is useful for when you end up on the members page. Such as when receiving a notification email about new members that contain a link to their profile page. There you don't have easy access to contact them. It takes more navigation to find it.

**Comment by mhall119** [comment]
If we only allow this when the user whose profile you  are looking at is a member of a team you admin/moderate, that would be okay. Making it available to everybody to send emails to users wouldn't be acceptable though. Would that work for you?

**Comment by ianp5a** [comment]
Yes.  The availability should stay the same as the existing email availability. No change to availability was intended.
